src/main/java/org/jabref/gui/desktop/os/Windows.java

```diff
@@ -27,13 +27,36 @@ public void openFile(String filePath, String fileType) throws IOException {
     @Override
     public String detectProgramPath(String programName, String directoryName) {
         String progFiles = System.getenv("ProgramFiles(x86)");
-        if (progFiles == null) {
-            progFiles = System.getenv("ProgramFiles");
+        String programPath;
+        if (progFiles != null) {
+            programPath = getProgramPath(programName, directoryName, progFiles);
+            if (programPath != null) {
+                return programPath;
+            }
         }
+
+        progFiles = System.getenv("ProgramFiles");
+        System.out.println(progFiles);
```

Comment:
Probably a left over from debugging. If you need debug output use LOGGER.debug(...)

Reply:
emmm，I was too careless. Thanks for your suggestion!